content/en/docs/setup/install/helm/index.md

```diff
@@ -10,8 +10,7 @@ test: no
 ---
 
 {{< warning >}}
-This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
-considered `pre-alpha`.
+This feature is currently considered alpha.
 {{< /warning >}}
 
```

Comment:
Why do you need a warning here? Usualluy we only warn about features that are pre-alpha, subject to major change.

Reply:
I see. I wanted to make it clear that Helm support is not beta yet so they can make installation choices based on all the data. How should I do that?

Reply:
Maybe just leave the sentence in the doc, but without the `{{< warning }}` wrapper. Warning seems a little too alarmist :)

Reply:
That's fair, I will update it.